src/app/components/AmountTextInput/index.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ *
+ * AmountTextInput
+ *
+ */
+import React from 'react'
+import { Box, Button, FormField, TextInput } from 'grommet'
+
+interface Props {
+  disabled: boolean
+  handleMaxValue: () => void
+  inline?: boolean
+  label?: string
+  max?: number
+  min?: number
+  placeholder?: string
+}
+
+const contentProps = {
+  border: { position: 'inner' },
+  round: 'xsmall',
+}
+
+export function AmountTextInput({
+  disabled,
+  handleMaxValue,
+  inline = false,
+  label,
+  max,
+  min = 0,
+  placeholder = '0',
+}: Props) {
+  return (
+    <>
+      <FormField
+        contentProps={inline ? contentProps : {}}
+        htmlFor="amount-id"
+        label={label}
+        name="amount"
+        validate={{
+          regexp: /^\d*((\.|,)\d{0,9})?$/,
```

Comment:
How many decimal places are allowed?  for example `AmountFormatter` defaults to 15, but it's not aligned with `10 ** 9`

Reply:
should be 9, dunno why AmountFormatter mentions 15

Reply:
Ok, thanks. I will reopen PR once it's ready.